data/kramdown-rfc2629.erb

```diff
@@ -16,8 +16,9 @@
 <?rfc <%=pi%>="<%= {true => "yes", false => "no", nil => "yes"}[val] || val %>"?>
 <% end -%>
 
-<rfc <%= ps.attrs("ipr", "docName=docname", "category=cat",
-                  "consensus", "submissionType=submissiontype", "xml:lang=lang",
+<% docname = ps[:docName] || ps[:docname] -%>
```

Comment:
I'll fix this by a ps retaining its attribute values.

Reply:
I pushed a cherry-pick (which also keeps docName optional, as v3.rnc specifies).